@netlify/plugin-functions-core: error message cuts package name

I am using `@prisma/client` in a function and put the dependency in the wrong `package.json` file (inside the function, instead of the root) which `@netlify/plugin-functions-core` does not like and outputs a helpful error message. Unfortunately the error message truncates the name of the package to `@prisma/c` in the first line:

```
...
7:58:26 PM: Packaging Functions from functions directory:
7:58:26 PM:  - image/image.js
7:58:26 PM: ​
7:58:26 PM: ┌────────────────────────────────────────────────────────┐
7:58:26 PM: │ Plugin "@netlify/plugin-functions-core" internal error │
7:58:26 PM: └────────────────────────────────────────────────────────┘
7:58:26 PM: ​
7:58:26 PM:   Error message
7:58:26 PM:   Error: A Netlify Function is using "@prisma/c" but that dependency has not been installed yet.
7:58:26 PM: ​
7:58:26 PM:   By default, dependencies inside a Netlify Function's "package.json" are not automatically installed.
  There are several ways to fix this problem:
7:58:26 PM:     - Removing your Function's "package.json" and adding the dependencies to the project's top-level "package.json" instead. This is the fastest and safest solution.
7:58:26 PM:     - Running "npm install" or "yarn" inside your Netlify Function in your build command.
7:58:26 PM:     - Adding the following plugin to your "netlify.toml":
7:58:26 PM: ​
7:58:26 PM:   [[plugins]]
7:58:26 PM:   package = "@netlify/plugin-functions-install-core"
7:58:26 PM: ​
7:58:26 PM:   In file "/opt/build/repo/functions/image/image.js": Cannot find module '@prisma/client'
7:58:26 PM:   Require stack:
7:58:26 PM:   - /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/node_modules/@netlify/zip-it-and-ship-it/src/resolve.js
7:58:26 PM:   - /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/node_modules/@netlify/zip-it-and-ship-it/src/dependencies.js
7:58:26 PM:   - /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/node_modules/@netlify/zip-it-and-ship-it/src/main.js
7:58:26 PM:   - /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/src/plugins_core/functions/plugin.js
7:58:26 PM:   - /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/src/plugins/child/logic.js
7:58:26 PM:   - /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/src/plugins/child/load.js
7:58:26 PM:   - /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/src/plugins/child/main.js
7:58:26 PM: ​
7:58:26 PM:   Plugin details
7:58:26 PM:   Package:        @netlify/plugin-functions-core
7:58:26 PM:   Version:        3.3.0
7:58:26 PM:   Repository:     git+https://github.com/netlify/build.git
7:58:26 PM:   npm link:       https://www.npmjs.com/package/@netlify/build
7:58:26 PM:   Report issues:  https://github.com/netlify/build/issues
7:58:26 PM: ​
7:58:26 PM:   Error location
7:58:26 PM:   In "onPostBuild" event in "@netlify/plugin-functions-core" from core
7:58:26 PM:       at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
7:58:26 PM:       at Function.resolve (internal/modules/cjs/helpers.js:83:19)
7:58:26 PM:       at resolvePathFollowSymlinks (/opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/node_modules/@netlify/zip-it-and-ship-it/src/resolve.js:62:18)
7:58:26 PM:       at resolvePackageFallback (/opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/node_modules/@netlify/zip-it-and-ship-it/src/resolve.js:74:24)
7:58:26 PM:       at resolvePackage (/opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/node_modules/@netlify/zip-it-and-ship-it/src/resolve.js:33:27)
7:58:26 PM: ​
7:58:26 PM:   Error properties
7:58:26 PM:   { code: 'MODULE_NOT_FOUND' }
7:58:26 PM: ​
7:58:26 PM:   Resolved config
7:58:26 PM:   build:
7:58:26 PM:     command: '#'
7:58:26 PM:     commandOrigin: ui
7:58:26 PM:     environment:
7:58:26 PM:       - DATABASE_URL
7:58:26 PM:     functions: /opt/build/repo/functions
7:58:27 PM: Caching artifacts
7:58:27 PM: Started saving build plugins
7:58:27 PM: Finished saving build plugins
7:58:27 PM: Started saving pip cache
7:58:27 PM: Finished saving pip cache
7:58:27 PM: Started saving emacs cask dependencies
7:58:27 PM: Finished saving emacs cask dependencies
7:58:27 PM: Started saving maven dependencies
7:58:27 PM: Finished saving maven dependencies
7:58:27 PM: Started saving boot dependencies
7:58:27 PM: Finished saving boot dependencies
7:58:27 PM: Started saving go dependencies
7:58:27 PM: Finished saving go dependencies
7:58:27 PM: Error running command: Build script returned non-zero exit code: 1
7:58:27 PM: Failing build: Failed to build site
7:58:27 PM: Failed during stage 'building site': Build script returned non-zero exit code: 1
7:58:27 PM: Finished processing build request in 14.46578208s
```

--- a/packages/build/src/plugins_core/functions/error.js
+++ b/packages/build/src/plugins_core/functions/error.js
@@ -57,7 +57,7 @@ const getModuleName = function(error) {
   return result[1]
 }
 
-const MODULE_NOT_FOUND_REGEXP = /Cannot find module '(@[^'/]+\/[^'/]|[^.'/][^'/]*)/
+const MODULE_NOT_FOUND_REGEXP = /Cannot find module '(@[^'/]+\/[^'/]+|[^.'/][^'/]*)/
 
 const getWarning = function(moduleName) {
   return `A Netlify Function is using "${moduleName}" but that dependency has not been installed yet.

